operator/v1/types.go

```diff
@@ -63,22 +63,22 @@ type HubCoreList struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:scope=Cluster
 
-// SpokeCore represents a deployment of nucleus core agent on spoke cluster.
-// When the deployment of spoke core agent is deployed, it will requires a secret
-// with the name of bootstrap-hub-kubeconfig in the namespace defined in the spec.
-type SpokeCore struct {
+// Klusterlet represents controllers on the managed cluster. When configured,
```

Comment:
I would go with parallel phrasing `Klusterlet configures the controllers on the...`